### PR TITLE
Updates to fieldsets and form columns

### PR DESCRIPTION
**Pull request by StevePole**

### CODE REVIEW

File: docs/examples/partials/activity-filters.html [hunk 1]
```diff
@@ -124,19 +124,19 @@
...
-		<div class="input-group input-group-sm">
-			<span class="input-group-addon">
-				<label class="control-label">From</label>
-			</span>
-			<input class="form-control" value="01 May 2016" />
-		</div>
+		<tw-date-lookup
+			placeholder="Start date..."
+			ng-model="$ctrl.startDate"
+			ng-max="$ctrl.endDate"
```
Comment (skywritergr) [inline docs/examples/partials/activity-filters.html:130]: ng-max for a date is probably not the most straightforward name for an interface. Could something like ng-maxDate be clearer?

File: docs/examples/partials/activity-filters.html [hunk 2]
```diff
@@ -124,19 +124,19 @@
...
-		<div class="input-group input-group-sm">
-			<span class="input-group-addon">
-				<label class="control-label">To</label>
-			</span>
-			<input class="form-control" value="22 May 2016" />
-		</div>
+		<tw-date-lookup
+			placeholder="End date..."
+			ng-model="$ctrl.endDate"
+			ng-min="$ctrl.startDate"
```
Comment (skywritergr) [inline docs/examples/partials/activity-filters.html:138]: same comment as above. Could we have a more descriptive name? ng-startDate?

**Review by skywritergr — APPROVED** [review]
Mostly alright, just some questions around the interface we use for the tw-date-lookup

File: docs/examples/partials/activity-filters.html (hunk 1 above)
Comment (StevePole) [reply docs/examples/partials/activity-filters.html:130]: This is implemented in styleguide components, just using it in the demo page here.  As I mentioned the other day, goal was to be compatible with the native and regular Angular interfaces: https://docs.angularjs.org/api/ng/input/input%5Bdate%5D

I am considering a change to allow different error messages to be used with the dynamic forms, but not quite ready for that.

image

**Pull request merged**